jpa/src/main/java/org/jboss/as/jpa/container/AbstractEntityManager.java

```diff
@@ -62,8 +62,6 @@
     public static final String PESSIMISTIC_WRITE_LOCK_MODE = "pessimistic_write";
     public static final String NONE_LOCK_MODE = "none";
 
-    private final transient boolean isTraceEnabled = ROOT_LOGGER.isTraceEnabled();
```

Comment:
If it's done like this the trace logging for this class won't work until the server is restarted or at least until a new instance is created.